client/themes/morning.css

```diff
@@ -208,17 +204,9 @@ body {
 }
 
 /* Embeds */
-#chat .toggle-content,
-#chat .toggle-button {
-	color: #f3f3f3;
-}
-
-#chat .toggle-button:hover {
-	color: rgba(243, 243, 243, .8);
-}
-
 #chat .toggle-content {
 	background: #242a33;
+	color: #f3f3f3;
 }
```

Comment:
All this makes the toggle button the same color than neighboring text. It was silly of me to color it differently, and it looked really bad on the condensed toggle.